cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/mutation/CVRMutationDataReader.java

```diff
@@ -192,10 +195,9 @@ private void annotateRecordsWithPOST(List<MutationRecord> records, boolean reann
         // records will be partitioned inside annotator client
         // records which do not get a response back will automatically be defaulted to an AnnotatedRecord(record)
         List<AnnotatedRecord> annotatedRecords = annotator.getAnnotatedRecordsUsingPOST(summaryStatistics, records, "mskcc", true, postIntervalSize, reannotate, "StripEntireSharedPrefix", Boolean.TRUE, Boolean.FALSE, Boolean.FALSE);
+        mutationRecords.addAll(annotatedRecords);
```

Comment:
Moved mutationRecords.add(ar); out of loop and called mutationRecords.addAll(annotatedRecords); instead.